src/Uno.UWP/Storage/Pickers/FileOpenPicker.wasm.cs

```diff
@@ -62,8 +63,9 @@ private async Task<FilePickerSelectedFilesArray> NativePickerPickFilesAsync(bool
 			var fileTypeAcceptTypes = BuildFileTypesMap();
 			var fileTypeAcceptTypesJson = JsonHelper.Serialize(fileTypeAcceptTypes);
 			var fileTypeMapParameter = WebAssemblyRuntime.EscapeJs(fileTypeAcceptTypesJson);
-
-			var nativeStorageItemInfosJson = await WebAssemblyRuntime.InvokeAsync($"{JsType}.nativePickFilesAsync({multipleParameter},{showAllEntryParameter},'{fileTypeMapParameter}')");
+			var id = WebAssemblyRuntime.EscapeJs(SettingsIdentifier ?? "");
```

Comment:
I think UWP throws for null `SettingsIdentifier` (I haven't confirmed that). If that's the case, we probably should match the same behavior.